src/de/jost_net/JVerein/gui/action/KontoDeleteAction.java

```diff
@@ -45,6 +48,16 @@ public void handleAction(Object context) throws ApplicationException
       {
         return;
       }
+      DBIterator<Buchung> it = Einstellungen.getDBService()
+          .createList(Buchung.class);
+      it.addFilter("konto = ?", new Object[] { k.getID() });
+      it.setLimit(1);
+      if (it.hasNext())
+      {
+        throw new ApplicationException(String.format(
+            "Konto '%s' kann nicht gel�scht werden. Es existiern Buchungen auf diesem Konto.",
+            k.getBezeichnung()));
+      }
```

Comment:
Anfangsbestand verweist auch auf das Konto und verhindert ein Löschen.
Da könnte man evtl. den Foreign Key in Anfangsbestand  von RESTRICT auf CASCADE ändern damit die implizit mit gelöscht werden. Sie werden ja ohne Konto nicht mehr gebraucht.

Reply:
Ja ich wollte di db Anpassungen in einem eigenen PR machen, da auch die aus #369